cas-client-core/src/main/java/org/jasig/cas/client/configuration/PropertiesConfigurationStrategyImpl.java

```diff
@@ -86,6 +88,18 @@ private boolean loadPropertiesFromFile(final String file) {
         if (CommonUtils.isEmpty(file)) {
             return false;
         }
+
+		if(file.startsWith(CLASSPATH_PREFIX)){
```

Comment:
oh,,, this code in my editor (intellij idea or notepad) , the code style is ok. but I don't know how can this happen.